package/MDAnalysis/analysis/helanal.py

```diff
@@ -215,10 +215,10 @@ def helanal_trajectory(universe, selection="name CA",
           If the specified finish time precedes the specified start time or
           current time stamp of trajectory object.
 
-    .. Note::
-
-       Only a single helix is analyzed. Use the selection to specify the
-       helix, e.g. with "name CA and resid 1:20" or use start=1, stop=20.
+    Note
```

Comment:
I think the numpy doc standard says it should be "Notes" (plural). https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#documenting-modules

Reply:
Thanks, fixed. (rebased and history rewritten)